pulser-core/pulser/devices/_device_datacls.py

```diff
@@ -746,48 +746,48 @@ class Device(BaseDevice):
     Attributes:
```

Comment:
Can we check out this looks with `Args:` instead of `Attributes:`? I think it might make the rendering more compact